pkgs/applications/virtualization/open-vm-tools/default.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation rec {
     repo   = "open-vm-tools";
     rev    = "stable-${version}";
     sha256 = "0idh8dqwb1df2di689090k9x1iap35jk3wg8yb1g70byichmscqb";
+    name   = "${pname}-${version}-source";
```

Comment:
```suggestion
```

the default name is intentional https://github.com/NixOS/nixpkgs/commit/c3255fe8ec326d2c8fe9462d49ed83aa64d3e68f

Reply:
My bad I put this in because I screwed up a fetch and needed to force my local env to redo it. Removed.